packages/crypto/src/crypto.ts

```diff
@@ -289,13 +297,13 @@ export const hpkeDecrypt = ({
  *
  * @param {string} credentialBundle - The encrypted credential bundle.
  * @param {string} embeddedKey - The private key for decryption.
- * @returns {Uint8Array} - The decrypted data or null if decryption fails.
+ * @returns {string} - The decrypted data or null if decryption fails.
  * @throws {Error} - If unable to decrypt the credential bundle
  */
-export const decryptBundle = (
+export const decryptEmailBundle = (
   credentialBundle: string,
   embeddedKey: string
-): Uint8Array => {
+): string => {
```

Comment:
Should we follow Arnaud's guidelines from a previous PR where the caller is to be responsible for decoding?

Reply:
Oh these are the higher level "helper" abstractions. hpkeDecrypt is the function that should return the uint8array

Reply:
oooh right right! disregard 👍